quick_setup_script.sh

```diff
@@ -12,6 +12,14 @@ echo aborting
 exit 0
 fi
 
+if [ -z ${MASTER_WALLET_PK+x} ]
+then
+echo "\$MASTER_WALLET_PK is empty"
+exit 0
+else
+echo "\$MASTER_WALLET_PK is NOT empty"
+fi
+
```

Comment:
pretty sure [this](https://github.com/teamsempo/SempoBlockchain/pull/202#issuecomment-607548767) was caused by MASTER_WALLET_PK being undefined... so added check

Reply:
oh nice!